### PR TITLE
Package SO: Create HTTSOP package for TPS2663xPWP

### PR DESCRIPTION
**Pull request by myfreescalewebpage**
Create HTTSOP package for TPS2663xPWP

Datasheet: https://www.ti.com/lit/ds/symlink/tps2663.pdf

Symbol PR: https://github.com/KiCad/kicad-symbols/pull/2686
Footprint PR: https://github.com/KiCad/kicad-footprints/pull/2260

f1

f2

### CODE REVIEW

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml [hunk 1]
```diff
@@ -267,6 +267,53 @@ Texas_PWP_R-PDSO-G20:
...
+    maximum: 0.30
+  lead_len:
+    minimum: 0.5
+    maximum: 0.75
+  pitch: 0.65
+  num_pins_x: 0
+  num_pins_y: 10
+  
+  EP_size_x: 3.4
+  EP_size_y: 6.5
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml:295]: Please use the actual package pad dimensions for `EP_size_[xy]` and EP_size_[xy]_overwrite for the final copper dimensions.

**Comment by myfreescalewebpage** [comment]
@chschlue what do you mean ? The EP is 3.4x6.5mm.

**Comment by myfreescalewebpage** [comment]
I have based the design on Texas_PWP_R-PDSO-G20 just above in the generator. http://www.ti.com/lit/ds/symlink/tlc5971.pdf#page=44

**Comment by chschlue** [comment]
The idea is to have the actual package dimensions captured in these size definitions (`EP_size_x: 2.16 .. 2.96`, `EP_size_y: 2.21 .. 2.96` in this case), followed by additional parameters for deviations from the script defaults (`EP_size_[xy]_overwrite` for actual copper dimensions and `EP_mask_[xy]` for actual mask cutout you want to achieve).

**Comment by myfreescalewebpage** [comment]
Understood, I have found an example in another yaml file. Will do it now.

**Comment by myfreescalewebpage** [comment]
I think this is correct now, I have generated and pushed again the footprint, as expected, no modification on the EP itself.

**Comment by chschlue** [comment]
FP merged.

**Pull request merged**